server/routes.ts

```diff
@@ -113,6 +113,12 @@ export default function routes(app: Express) {
   router.route('/workspaces/:id').delete(isAuth('admin'), workspace.delete);
```

Comment:
look like upper we have same problem every auth user can get all workspace, that not what we want

Reply:
Updated PR with fix in commit 6269884